chia/consensus/multiprocess_validation.py

```diff
@@ -147,21 +146,21 @@ def pre_validate_block(
         return PreValidationResult(uint16(Err.UNKNOWN.value), None, None, uint32(validation_time * 1000))
 
 
-async def pre_validate_blocks_multiprocessing(
+async def pre_validate_block(
     constants: ConsensusConstants,
     blockchain: AugmentedBlockchain,
-    blocks: Sequence[FullBlock],
+    block: FullBlock,
     pool: Executor,
-    block_height_conds_map: dict[uint32, SpendBundleConditions],
+    conds: Optional[SpendBundleConditions],
```

Comment:
when validating a block whose `UnfinishedBlock` counterpart has been validated, we already have the `SpendBundleConditions`, so we save time by passing that in here